test/stdlib/CodableTests.swift

```diff
@@ -845,6 +986,13 @@ class TestCodable : TestCodableSuper {
             expectRoundTripEqualityThroughPlist(for: url, lineNumber: testLine)
         }
     }
+    
+    func test_URL_JSON_Errors() {
+        expectDecodingErrorViaJSON(
+            type: URL.self,
+            json: "http://swîft.org",
```

Comment:
I'm a bit wary of this test. This URL is a valid URL (at least with modern versions of Foundation), but I think this fails only because it's not surrounded by quotes (or perhaps on older OS versions it fails because we didn't IDNA encode the URL). Perhaps a meta question here: should the stdlib have tests like this that test Foundation Codable conformances? That seems likely to cause issues if Foundation changes independently. It might be best to add this type of test to Foundation instead of here so we can update the test if parsing behavior changes

Reply:
Makes sense! I'll look at moving this one over to the Foundation project